packages/paima-sdk/paima-utils/src/config/loading.ts

```diff
@@ -131,6 +133,16 @@ export const AvailConfigSchema = Type.Intersect([AvailRequiredProperties, AvailO
 export type AvailMainConfig = AvailConfig & { type: ConfigNetworkType.AVAIL_MAIN };
 export type AvailConfig = Static<typeof AvailConfigSchema>;
 
+export const MidnightConfigSchema = Type.Object({
+  // Midnight network connection info.
+  networkId: Type.Number(),
+  indexer: Type.String(),
+  indexerWS: Type.Optional(Type.String()),
+  // node URL and proof server are not needed for our read-only use
+});
+
+export type MidnightConfig = Static<typeof MidnightConfigSchema>;
```

Comment:
```suggestion
export const BaseMidnightConfigSchema = Type.Object({
  // Midnight network connection info.
  networkId: Type.Number(),
  indexer: Type.String(),
  indexerWS: Type.Optional(Type.String()),
  // node URL and proof server are not needed for our read-only use
});

export const MidnightConfigSchema = addType(BaseMidnightConfigSchema , ConfigNetworkType.MIDNIGHT);

export type MidnightConfig = Static<typeof MidnightConfigSchema>;
```
based on the changed required for #430